### PR TITLE
Implement vtx_akk_hack cli command as a switchable feature

### PR DESCRIPTION
**Pull request by LexioTech**
- Refactored the current CLI based implementation of vtx_akk_hack into a new feature named: "LEGACY_SA_SUPPORT"

- Re-added the tailing byte in send frame as a possible fix for the "standard" implementation of Smart Audio, due to user reporting that turning the hack off for Unify and other "standard" SA VTX-es will result in an incorrect setting save issue (settings are only saved after unplugging and re-plugging the battery)

- This PR should be accepted with the [relevant Butterflight-Configurator PR](https://github.com/ButterFlight/butterflight-configurator/pull/17) that adds the "LEGACY_SA_SUPPORT" feature to the configuration tab

- Incrementing the MSP protocol minor version to 39 and the EEPROM version is required before release

### CODE REVIEW

File: src/main/config/feature.h [hunk 1]
```diff
@@ -51,6 +51,7 @@ typedef enum {
     FEATURE_ESC_SENSOR = 1 << 27,
     FEATURE_ANTI_GRAVITY = 1 << 28,
     FEATURE_DYNAMIC_FILTER = 1 << 29,
+	FEATURE_LEGACY_SA_SUPPORT = 1 << 30,
```
Comment (kidBrazil) [inline src/main/config/feature.h]: I know its merely cosmetic but can we line it up ?

File: src/main/interface/cli.c [hunk 2]
```diff
@@ -184,7 +184,7 @@ static const char * const featureNames[] = {
     "RANGEFINDER", "TELEMETRY", "", "3D", "RX_PARALLEL_PWM",
     "RX_MSP", "RSSI_ADC", "LED_STRIP", "DISPLAY", "OSD",
     "", "CHANNEL_FORWARDING", "TRANSPONDER", "AIRMODE",
-    "", "", "RX_SPI", "SOFTSPI", "ESC_SENSOR", "ANTI_GRAVITY", "DYNAMIC_FILTER", NULL
+    "", "", "RX_SPI", "SOFTSPI", "ESC_SENSOR", "ANTI_GRAVITY", "DYNAMIC_FILTER", "LEGACY_SA_SUPPORT", NULL
```
Comment (kidBrazil) [inline src/main/interface/cli.c]: I dig the name

File: src/main/io/vtx_smartaudio.c [hunk 3]
```diff
@@ -447,9 +454,8 @@ static void saSendFrame(uint8_t *buf, int len)
         serialWrite(smartAudioSerialPort, buf[i]);
     }
 
-    if (vtxSettingsConfig()->akkStyleEndFrame) {
-      serialWrite(smartAudioSerialPort, 0x00); // Added back to fix old AKK - breaks custom freq.
-    }
+	serialWrite(smartAudioSerialPort, 0x00); // Re-added for both "Legacy" and "Standard" SA implementations
```
Comment (kidBrazil) [inline src/main/io/vtx_smartaudio.c]: I am really hoping this is going to solve some of that weirdness.. though we should look more into the software pulldown.

File: src/main/config/feature.h (hunk 1 above)
Comment (LexioTech) [reply src/main/config/feature.h]: Done, sorry, Visual Studio generates a tab from 3 spaces by default -.-, turned it off.

**Pull request merged**